tools/generators/bazel_dependencies/src/Generator/Calculate.swift

```diff
@@ -0,0 +1,166 @@
+import PBXProj
+
+extension Generator {
+    /// Calculates the BazelDependencies `PBXProj` partial.
+    static func calculate(
+        buildSettings: String,
+        defaultXcodeConfiguration: String?,
+        postBuildRunScript: String?,
+        preBuildRunScript: String?,
+        xcodeConfigurations: [String]
+    ) -> String {
+        // Build phases
+
+        var buildPhases: [(id: String, element: String)] = []
+
+        if let preBuildRunScript = preBuildRunScript {
+            let id = Identifiers.BazelDependencies.preBuildScript
+            buildPhases.append((
+                id: id,
+                element: #"""
+		\#(id) = \#(preBuildRunScript);
+"""#
+            ))
+        }
+
+        buildPhases.append((
+            id: Identifiers.BazelDependencies.bazelBuild,
+            element: #"""
+		\#(Identifiers.BazelDependencies.bazelBuild) = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
```

Comment:
I assume this is always the same and that's why we can hard code it like this right?

Reply:
Yeah: https://github.com/tuist/XcodeProj/blob/f570155209af12643309ac4e758b875c63dcbf50/Sources/XcodeProj/Objects/BuildPhase/PBXBuildPhase.swift#L6